Lib/fontbakery/profiles/googlefonts.py

```diff
@@ -3791,6 +3792,83 @@ def com_google_fonts_check_name_family_and_style_max_length(ttFont):
     yield PASS, "All name entries are good."
 
 
+@check(
+    id='com.google.fonts/check/has_unacceptable_control_chars',
+    conditions=['are_ttf']
+)
+def com_google_fonts_check_has_unacceptable_control_chars(ttFonts):
+  """Does font file include unacceptable control character glyphs?
+
+  Args:
+      ttFonts (list): list of one or more fontTools.ttLib.TTFont objects
```

Comment:
This is valid for all family-wide fontbakery checks, and similar for all the single-file checks. Please remove the args description as it diverges from the docstring style we're using in the rest of the code.

Reply:
done!